Sources/Clappr_iOS/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -120,7 +121,7 @@ open class AVFoundationPlayback: Playback {
         }
 
         if playbackType == .live {
-            if supportDVR, let duration = seekableTimeRanges.first?.timeRangeValue.duration.seconds {
+            if isDvrAvailable, let duration = seekableTimeRanges.first?.timeRangeValue.duration.seconds {
```

Comment:
About the duration, we should be careful about considering the duration of only the first seekable range. If there's a discontinuity between ranges, we are discarding the other ranges from this duration, so it would report a smaller value than it should be. The right way of reporting this is to use the duration of all seekable time ranges.

Reply:
Thanks for the feedback Towerz. Is it possible to get a playlist with this discontinuity scenario to see how the seekableTimeRanges changes?

In all our testing scenarios, the first item of the array was the only one returning a value, that’s why we kept only one Time Range.

Reply:
@nabilsafatli sure, I'm arranging to get a live video simulating this kind of behavior. As soon as I have this ready I'll let you know.